message/validation/partial_validation.go

```diff
@@ -112,6 +118,8 @@ func (mv *messageValidator) validatePartialSignatureMessageSemantics(
 			return err
 		}
 
+		// MSGVALIDATIONREVIEW: since we don't expect operators to be synced on the committees' validators set,
```

Comment:
Being fixed here: https://github.com/ssvlabs/ssv/pull/1488

Reply:
I'm not sure I agree because when there are actions with a share, we update the state, so I don't understand why we don't expect this. Could you please add more details?

Reply:
This may be for the following case. Let's say we have operators A and B. Operator A is part of Committee C, which just added a new validator V.

For our current state, we can't assume that operator B must be up to date with this information. So, whenever A sends a partial signature message on behalf of committee C that includes the index of validator V, B may not be aware of this (so it can't validate).

Even in the protocol (in-committee), when processing partial signature messages, we simply do "continue" in the iteration if we don't know the validator index